electron/src/runtime/EnvironmentUtil.ts

```diff
@@ -42,7 +42,7 @@ enum BackendType {
   RC = 'RC',
 }
 
-enum BackendTypeLabel {
+export enum BackendTypeLabel {
```

Comment:
What is the difference with the previous version of the export?

Reply:
The previous one didn't compile with `@babel/preset-typescript`.